java/flight/flight-core/src/main/java/org/apache/arrow/flight/auth2/ClientIncomingAuthHeaderMiddleware.java

```diff
@@ -17,40 +17,52 @@
 
 package org.apache.arrow.flight.auth2;
 
-import java.util.concurrent.atomic.AtomicReference;
-
 import org.apache.arrow.flight.CallHeaders;
 import org.apache.arrow.flight.CallInfo;
 import org.apache.arrow.flight.CallStatus;
 import org.apache.arrow.flight.FlightClientMiddleware;
+import org.apache.arrow.flight.grpc.CredentialCallOption;
+import org.slf4j.Logger;
```

Comment:
I deleted these from the original class. You should rebase on the top of the flight-auth-redesign branch to make sure you are up to date.

Reply:
Done, removed. I believe I added these back during a merge conflict in the rebase.